pyproject.toml

```diff
@@ -24,7 +24,7 @@ classifiers = [
 ]
 dependencies = [
     "docstring_parser>=0.16",  # Pin for description addition
-    "httpx",
+    "httpx < 0.28",  # Pin until we remove app argument in clients that was deprecated in 0.28
```

Comment:
Can you do this pinning in the `dev` extra so we don't require it for core deps, as it's really a dev-time issue